index.html

```diff
@@ -926,26 +925,26 @@
     </p>
     <ul>
       <li>
-        Only <a>browsing context</a>s with a defined <a>origin</a>, whose schema
-        is starting with <code>"https"</code> must be able to access
+        Only <a>script execution environment</a>s of <code>
+        <a href="https://html.spec.whatwg.org/#document">Document</a></code>s
+        with a <a>browsing context</a>, and an <a>incumbent settings object</a>
+        which is a <a>secure context</a> are allowed to access
         <a>NFC content</a>.
         Browsers may ignore this rule for development purposes only.
       </li>
       <li>
-        In order to use NFC, the top-level <a>browsing context</a> of the
-        <code>Window</code> object using the Web NFC API must be
-        <strong>visible</strong> and <strong>in focus</strong>. This also means
-        that UAs should make sure the display is on, and the device is unlocked.
+        In order to use NFC, the <code>Window</code> object associated with the
+        <code><a>Document</a></code> contained by the
+        <a href="http://www.w3.org/TR/html5/browsers.html#top-level-browsing-context">
+        top level browsing context</a> using
+        the Web NFC API must be
+        <a href="http://www.w3.org/TR/page-visibility/#now-visible-algorithm">
+        visible</a> and in
+        <a href="https://html.spec.whatwg.org/#focus">focus</a>.
+        This also means that UAs should make sure the display is on, and the
```

Comment:
"make sure the display is on and the device is unlocked" could mean that the UA should turn on the display and unlock the device. Instead, perhaps "This also means that UAs should block access to the NFC radio if the display is off or the device is locked."